c10/core/Scalar.h

```diff
@@ -140,6 +140,7 @@ class C10_API Scalar {
     // Can't do put std::complex in the union, because it triggers
     // an nvcc bug:
     //    error: designator may not specify a non-POD subobject
+    // TODO: can we put c10::complex to it?
```

Comment:
I think it is helpful to have a concrete action on the TODO or else discuss it on an issue if there are some blockers.

Reply:
Can we keep this line here as is, and I will do some experiment with it and decide whether to add an issue or just make the change.